packages/pg/package.json

```diff
@@ -11,43 +11,55 @@
     "postgresql",
     "rdbms"
   ],
-  "homepage": "https://github.com/brianc/node-postgres",
+  "homepage": "git://github.com/daniel-liu-bitio/node-postgres.git",
   "repository": {
     "type": "git",
-    "url": "git://github.com/brianc/node-postgres.git",
+    "url": "git://github.com/daniel-liu-bitio/node-postgres.git",
     "directory": "packages/pg"
   },
-  "author": "Brian Carlson <brian.m.carlson@gmail.com>",
-  "main": "./lib",
-  "dependencies": {
-    "buffer-writer": "2.0.0",
-    "packet-reader": "1.0.0",
-    "pg-connection-string": "^2.5.0",
-    "pg-pool": "^3.5.1",
-    "pg-protocol": "^1.5.0",
-    "pg-types": "^2.1.0",
-    "pgpass": "1.x"
+  "author": "Daniel Liu",
+  "main": "./dist/webpack-bundle.js",
```

Comment:
What's the resulting bundle size and how do we plan to server this all to clients? Can it be broken down into smaller modules by nextjs and served?

Reply:
Bundle size is around 1MB now, most (~50%) is coming from npm packages involved with encryption